src/org/parosproxy/paros/network/SSLConnector.java

```diff
@@ -572,6 +573,9 @@ public void decorate(SSLSocket sslSocket) {
 				readSupportedProtocols(sslSocket);
 			}
 			sslSocket.setEnabledProtocols(getClientEnabledProtocols());
+			if (!relaxedTrust) {

```

Comment:
Fixed, now matches what we talked about back here:
 * kingthorin https://github.com/zaproxy/zaproxy/pull/1977#issuecomment-148732399
 * thc202 https://github.com/zaproxy/zaproxy/pull/1977#issuecomment-148734499